common/test/acceptance/tests/video/test_video_events.py

```diff
@@ -47,33 +94,193 @@ def is_video_event(event):
                 assert_event_matches({'event_type': 'pause_video'}, video_event)
                 self.assert_valid_control_event_at_time(video_event, self.video.seconds)
 
-    def assert_payload_contains_ids(self, video_event):
+    def test_strict_event_format(self):
         """
-        Video events should all contain "id" and "code" attributes in their payload.
+        This test makes a very strong assertion about the fields present in events. The goal of it is to ensure that new
+        fields are not added to all events mistakenly. It should be the only existing test that is updated when new top
+        level fields are added to all events.
+        """
+
+        captured_events = []
+        with self.capture_events(lambda e: e['event_type'] == 'load_video', captured_events=captured_events):
+            self.navigate_to_video()
+
+        load_video_event = captured_events[0]
+
+        # Validate the event payload
+        self.assert_payload_contains_ids(load_video_event)
+
+        # We cannot predict the value of these fields so we make weaker assertions about them
+        dynamic_string_fields = (
+            'accept_language',
+            'agent',
+            'host',
+            'ip',
+            'event',
+            'session'
+        )
+        for field in dynamic_string_fields:
+            self.assert_field_type(load_video_event, field, basestring)
+            self.assertIn(field, load_video_event, '{0} not found in the root of the event'.format(field))
+            del load_video_event[field]
+
+        # A weak assertion for the timestamp as well
+        self.assert_field_type(load_video_event, 'time', datetime.datetime)
+        del load_video_event['time']
+
+        # Note that all unpredictable fields have been deleted from the event at this point
+
+        course_key = CourseKey.from_string(self.course_id)
+        static_fields_pattern = {
+            'context': {
+                'course_id': unicode(course_key),
+                'org_id': course_key.org,
+                'path': '/event',
+                'user_id': self.user_info['user_id']
+            },
+            'event_source': 'browser',
+            'event_type': 'load_video',
+            'username': self.user_info['username'],
+            'page': self.browser.current_url,
+            'referer': self.browser.current_url,
+            'name': 'load_video',
+        }
+        assert_events_equal(static_fields_pattern, load_video_event)
+
+
+@ddt.ddt
+class VideoBumperEventsTest(VideoEventsTestMixin):
+    """ Test bumper video event emission """
+
+    # helper methods
+    def seek_video_and_skip(self):
```

Comment:
nit: I got thrown by the names of these -- a "seek" made me think this would be jumping to a given spot, rather than watching until a given spot.  Would this be more accurate as "watch_video_and_skip" (or _dismiss)?